room/compiler/src/test/kotlin/androidx/room/processor/DaoProcessorTest.kt

```diff
@@ -211,6 +211,23 @@ class DaoProcessorTest(private val enableVerification: Boolean) {
         }
     }
 
+    @Test
+    fun skipQueryVerificationPojo() {
```

Comment:
Can you please move this test to `QueryMethodProcessorTest` since that is more closer to where the behaviour is changing rather than the `DaoProcessor`. The test method `singleQueryMethod` should also help you further very this by asserting the result binder and its row adapter to be a PojoAdapter.